src/dapi-names.ts

```diff
@@ -0,0 +1,38 @@
+import { Client } from 'pg';
+import { ethers } from 'ethers';
+import { go } from '@api3/promise-utils';
+import { goQueryConfig } from './constants';
+
+export const dapiNameQueryTemplate = `
```

Comment:
How to test this locally? 

It would be nice to have some test DB because now it's not possible to test/deploy this on it's own. However, this should be done in a separate PR.

Reply:
Yes, ideally one should probably start a test DB in the CI system maybe.

Reply:
Localhost would be enough imo. Currently, you are the only one who is able to conveniently run this :/

Reply:
I plan on storing the latest deployment files in Keybase, which anyone with access will be able to use to redeploy and/or test (with production data).
But I totally agree with you that we need a test DB :)